build.sbt

```diff
@@ -24,7 +24,7 @@ lazy val commonSettings = Seq(
   organizationName := "Cognite",
   organizationHomepage := Some(url("https://cognite.com")),
   version := "2.5.14",
-  isSnapshot := true,
+  // isSnapshot := true,
```

Comment:
I'd recommend not using snapshot at all (or by default) and use automatically generated version based on commit history depth, aka reapply https://github.com/cognitedata/cognite-sdk-scala/commit/4497505cca8a580f1a4a8ba1f3fedf29da65fcaf

Reply:
agree, this shouldn't be so manual..... at the very least, some pre-commit check could prevent the mistakes that lead to need 4 PRs to change one line and publish 😆